### ISSUE TITLE
WebAppDIRAC v4r2p6 broken

### ISSUE THREAD

**Issue by chrisburr** [DIRACGrid/WebAppDIRAC#517]
The new downtime stuff caused latest update in LHCb to fail. Initially it failed due to `GLOBAL.APP.applicationInDowntime` being undefined (perhaps a caching issue). After that went away it still fails with:

```
Uncaught TypeError: Date.now is not a constructor
``` 

For now I've just disabled it with a hotfix:

```diff
--- a/WebAppDIRAC/WebApp/static/core/js/views/tabs/Main.js.bak
+++ b/WebAppDIRAC/WebApp/static/core/js/views/tabs/Main.js
@@ -808,10 +808,10 @@ Ext.define("Ext.dirac.views.tabs.Main", {
     var me = this;

     // Do not create new window if the application in downtime
-    var message = GLOBAL.APP.applicationInDowntime(moduleName);
-    if (message) {
-      return GLOBAL.APP.CF.alert(message, "info");
-    }
+    //var message = GLOBAL.APP.applicationInDowntime(moduleName);
+    //if (message) {
+    //  return GLOBAL.APP.CF.alert(message, "info");
+    //}

     Ext.get("app-dirac-loading").show();
```

**Comment by chrisburr** [comment]
Related, was the a reason for adding this to a production release instead of targetting integration?

**Comment by TaykYoku** [comment]
This feature is added on request by this issue #395 that targeted to v4r2.

**Comment by TaykYoku** [comment]
These bugs have been fixed here #522, #520

**Comment by TaykYoku** [comment]
Fixed in  WebApp v4r2p8
